cumulus/pallets/xcmp-queue/Cargo.toml

```diff
@@ -38,7 +38,7 @@ pallet-balances = { path = "../../../substrate/frame/balances" }
 xcm-builder = { package = "staging-xcm-builder", path = "../../../polkadot/xcm/xcm-builder" }
 
 # Cumulus
-cumulus-pallet-parachain-system = { path = "../parachain-system" }
+cumulus-pallet-parachain-system = { path = "../parachain-system", features = ["parameterized-consensus-hook"] }
```

Comment:
Why did you do this?

Reply:
It is being enabled everywhere, you can search for `features = ["parameterized-consensus-hook"` in the codebase...  
I asked about this in [elements](https://matrix.to/#/!aJymqQYtCjjqImFLSb:parity.io/$4eTJjxGA07oNt3UK_FLJnO-z44lksRnROI9jAhHi7C4?via=parity.io&via=web3.foundation&via=matrix.org) as well:
<img width="426" alt="Screenshot 2023-09-07 at 21 58 48" src="https://github.com/paritytech/polkadot-sdk/assets/10380170/31452869-b808-4ec1-aca5-42f5a1b14c6e">

Reply:
@ggwpez hmm, but `cargo test --release -p cumulus-pallet-xcmp-queue` does not work without that,
looks like this is the only place where `cumulus-pallet-parachain-system` is used as **only** `dev-dependencies`,

another place is [aura-ext](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/pallets/aura-ext/Cargo.toml#L22-L28), but there is a little bit different situation - `cumulus-pallet-parachain-system` is there twice as dev and also as non-dev dependency
```
# Cumulus
cumulus-pallet-parachain-system = { path = "../parachain-system", default-features = false }

[dev-dependencies]

# Cumulus
cumulus-pallet-parachain-system = { path = "../parachain-system" }
```

Reply:
@ggwpez @bkchr 
guys, please, any conclusion here? 

Oliver, I see you've already added the same [here](https://github.com/paritytech/polkadot-sdk/pull/1246/files#diff-4d696877d5f81afd1ada4c09c7d27e734f8518a85b9e563026d9d2d702de7795R42)